pkg/statsd/handler_http_forwarder_v2.go

```diff
@@ -139,6 +143,24 @@ func NewHttpForwarderHandlerV2(
 		headers[k] = v
 	}
 
+	dynHeaderNamesWithColon := make([]string, 0)
+	for _, name := range dynHeaderNames {
+		if name == "" {
+			continue
+		}
+		isCustomHeader := false
+		for xk := range xheaders {
```

Comment:
This can be done as...
```
if _, ok := xheaders[name]; !ok {
  dynHeaderNamesWithColon = append(dynHeaderNamesWithColon, name+":")
} else {
  logger.WithField("header-name", name).Warn("static and dynamic header defined, using static")
}
```

Reply:
Thanks! Updated as suggested.